packages/grid/test/selection.test.js

```diff
@@ -509,4 +510,52 @@ describe('multi selection column', () => {
 
     expect(grid.selectedItems).to.eql(grid.items);
   });
+
+  it('should select items on mouse drag when selectRowsByDragging is enabled', () => {
```

Comment:
Let's wrap the new tests into a nested suite (`describe`) for select rows by dragging.

Reply:
Done